data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelProtoCodec.java

```diff
@@ -198,17 +193,6 @@ protected Collection<OpenTelemetryLog> parseResourceLogs(ResourceLogs rs, final
             final Map<String, Object> resourceAttributes = OTelProtoCodec.getResourceAttributes(rs.getResource());
             final String schemaUrl = rs.getSchemaUrl();
 
-            Stream<OpenTelemetryLog> mappedInstrumentationLibraryLogs = rs.getInstrumentationLibraryLogsList()
```

Comment:
Why are these removed and not changed to iterate of `getScopeLogs()`?

Reply:
scopeLogs is being processed in ` rs.getScopeLogsList().stream()` below